dali/pipeline/operators/crop/crop.cu

```diff
@@ -90,10 +90,22 @@ void Crop<GPUBackend>::RunHelper(Workspace<GPUBackend> *ws, const int idx) {
 template <>
 void Crop<GPUBackend>::SetupSharedSampleParams(DeviceWorkspace *ws) {
   const auto &input = ws->Input<GPUBackend>(0);
+    int total_batch = batch_size_ * SequenceSize(0);
+  if (SequenceSize(0) > 1) {
+    Init(total_batch);
+    std::vector<int> new_crop_height_(total_batch);
+    std::vector<int> new_crop_width_(total_batch);
+    for (int i = 0; i < total_batch; ++i) {
+      new_crop_height_[i] = crop_height_[i / SequenceSize(0)];
```

Comment:
you have many calls to SequenceSize(0). You could cache it in a variable at the beginning of the method

Reply:
Done

Reply:
Compiler should do it for you.